src/Library/demos/Box/main.blp

```diff
@@ -0,0 +1,202 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.Clamp {
+  maximum-size: 1024;
+
+  Box {
+    orientation: vertical;
+
+    Label {
+      label: "Box";
+      margin-top: 12;
+      margin-bottom: 12;
+      styles [
+        "title-1",
+      ]
+    }
+
+    Label {
+      label: _("A widget that arranges its child widgets into a single row or column");
+    }
+
+    Box{
+      halign: center;
+      valign: end;
+      margin-bottom: 12;
+
+      LinkButton{
+        label: "Tutorial";
+        uri: "https://developer.gnome.org/documentation/tutorials/beginners/components/box.html";
+      }
+
+      LinkButton{
+        label: "API Reference";
+        uri: "https://docs.gtk.org/gtk4/class.Box.html";
+      }
+    }
+
+    Box {
+      orientation: vertical;
+
+      Box {
+        hexpand: true;
+        orientation: vertical;
+        spacing: 12;
+        margin-bottom:24;
+
+        Box {
+          halign: center;
+          spacing: 18;
+
+          Button button_append{
+            label: _("Append Item");
+            styles [
+              "pill",
+            ]
+          }
+
+          Button button_prepend{
+            label: _("Prepend Item");
+            styles [
+              "pill",
+            ]
+          }
+
+          Button button_remove{
+            label: _("Remove Item");
+            styles [
+              "pill",
+            ]
+          }
+        }
+      }
+
+        Box {
+          halign:center;
+          spacing:18;
+
+          Box{
+            Label {
+              label: _("orientation");
+            }
+
+            Box {
+              margin-start: 6;
+              homogeneous: true;
+              halign: center;
+
+              ToggleButton toggle_orientation_horizontal {
+                label: _("horizontal");
+                active: true;
+              }
+              ToggleButton toggle_orientation_vertical {
+                label: _("vertical");
+                group: toggle_orientation_horizontal;
+              }
```

Comment:
This is a perfect time to use the `linked` class, which will join the two buttons together as a visibly logical unit :slightly_smiling_face:

Reply:
Ah my bad, I might have accidentally removed it :/